README.adoc

```diff
@@ -253,10 +314,11 @@ $ python travel.py
 $ Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
 ----
```

Comment:
Double check instructions. When I viewed the file, I saw duplicate instructions under Python instruction guide

Reply:
Updated